chroma-agent/chroma_agent/agent_daemon.py

```diff
@@ -207,4 +208,9 @@ def teardown_callback(*args, **kwargs):
         # NB I would rather ensure cleanup by using 'with', but this
         # is python 2.4-compatible code
         context.close()
+
+    # Call any agent daemon startup methods that were registered.
+    for function in agent_daemon_teardown_functions:
```

Comment:
What happens on a `SIGINT`? I see there is a `teardown_callback`, do we want to add the cleanup there as well?